exporters/deserializers/__init__.py

```diff
@@ -0,0 +1,25 @@
+from exporters.pipeline.base_pipeline_item import BasePipelineItem
```

Comment:
Nitpicking: how about moving this file into a `deserializers.py` in the upper-level?

I prefer not to have much code in `__init__.py`, it's sometimes hard to find it. :)

Same about decompressors module.